lib/src/participant/remote.dart

```diff
@@ -190,7 +190,8 @@ class RemoteParticipant extends Participant<RemoteTrackPublication> {
   }
 
   @override
-  Future<void> unpublishTrack(String trackSid, {bool notify = true}) async {
+  Future<void> unpublishTrack(String trackSid,
+      {bool notify = true, bool? stopOnUnpublish}) async {
```

Comment:
`stopOnUnpublish` is unused. we need to avoid calling `track.stop()` when this is true.

Reply:
In fact, I am a little confused here, unpublishTrack actually operates RemoteTrack, and the function name sounds like it should control the publishing track.

Reply:
seems to be changed to 
`Future<void> unSubscribeRemoteTrack(String trackSid, {bool notify = true, bool? stopOnUnSubscribe});` 
more reasonable?

Reply:
oh you are right, the naming is quite confusing here. it should be something like `removePublishedTrack(String trackSid, {bool notify = true})`

Reply:
or maybe `removePublication`

Reply:
https://github.com/livekit/client-sdk-flutter/pull/194/commits/18f94d03ace109964b84702f276814e26b5922fd